spec/api-browser-window-spec.js

```diff
@@ -1409,6 +1409,8 @@ describe('BrowserWindow module', () => {
             preload: preload
           }
         })
+        ipcRenderer.send('set-web-preferences-on-next-new-window', w.webContents.id, 'preload', preload)
```

Comment:
Where is this IPC handled?

Reply:
other tests are already using this IPC to configure popups

Reply:
in `spec/static/main.js`
```
ipcMain.on('set-web-preferences-on-next-new-window', (event, id, key, value) => {
  webContents.fromId(id).once('new-window', (event, url, frameName, disposition, options) => {
    options.webPreferences[key] = value
  })
})
```

Reply:
NVM I see now: https://github.com/electron/electron/blob/master/spec/static/main.js#L285